migrations/20200513_01_kPWNp-create-base-tables.py

```diff
@@ -0,0 +1,56 @@
+# 20200513_01_kPWNp-create-base-tables.py
+
+from yoyo import step
+
+__depends__ = {}
+
+steps = [
+    step(
+        'CREATE TYPE team_method AS ENUM(\'captains\', \'autobalance\', \'random\');',
+        'DROP TYPE team_method;'
+    ),
+    step(
+        'CREATE TYPE captain_method AS ENUM(\'volunteer\', \'rank\', \'random\');',
+        'DROP TYPE captain_method;'
+    ),
+    step(
+        (
+            'CREATE TABLE guilds(\n'
```

Comment:
Could you use a multi line string here?

Reply:
That looks like a multi-line string to me?